server/src/main/java/org/elasticsearch/inference/ModelConfigurations.java

```diff
@@ -28,8 +28,13 @@ public class ModelConfigurations implements ToFilteredXContentObject, VersionedN
     public static final String USE_ID_FOR_INDEX = "for_index";
     public static final String SERVICE = "service";
     public static final String SERVICE_SETTINGS = "service_settings";
-    public static final String TASK_SETTINGS = "task_settings";
+    public static final String OLD_TASK_SETTINGS = "task_settings";
```

Comment:
Why are these now `OLD_TASK_SETTINGS`? Do we need to do this rename if there are no new task settings?

Reply:
We don't need to rename it, I'll revert the rename.